front/src/modules/settings/data-model/components/SettingsObjectFieldTypeSelectSection.tsx

```diff
@@ -127,7 +127,10 @@ export const SettingsObjectFieldTypeSelectSection = ({
             form={
               values.type === FieldMetadataType.Relation && (
                 <SettingsObjectFieldRelationForm
-                  disableRelationEdition={!!relationFieldMetadataId}
+                  disableFieldEdition={
+                    !!relationFieldMetadata && !relationFieldMetadata.isCustom
+                  }
+                  disableRelationEdition={!!relationFieldMetadata?.id}
```

Comment:
would something like isNotEmpty(relationFieldMetadata) work? I'm a bit confused by the fact we check the id specifically